### ISSUE TITLE
System Log Errors since last update

### ISSUE THREAD

**Issue by BondBug** [oziee/ha-solcast-solar#172]
### The problem

Logger: custom_components.solcast_solar.solcastapi
Source: custom_components/solcast_solar/solcastapi.py:196
Integration: Solcast PV Forecast ([documentation](https://github.com/oziee/ha-solcast-solar), [issues](https://github.com/oziee/ha-solcast-solar/issues))
First occurred: 07:53:16 (16 occurrences)
Last logged: 07:53:18

not using item {'period_end': '2023-08-25T03:00:00.0000000Z', 'pv_estimate': 0, 'pv_estimate10': 0, 'pv_estimate90': 0}
not using item {'period_end': '2023-08-25T03:30:00.0000000Z', 'pv_estimate': 0, 'pv_estimate10': 0, 'pv_estimate90': 0}
not using item {'period_end': '2023-08-25T05:00:00.0000000Z', 'pv_estimate': 0, 'pv_estimate10': 0, 'pv_estimate90': 0}
not using item {'period_end': '2023-08-25T05:30:00.0000000Z', 'pv_estimate': 0.0088, 'pv_estimate10': 0.0015, 'pv_estimate90': 0.0099}
not using item {'period_end': '2023-08-25T07:00:00.0000000Z', 'pv_estimate': 0.2092, 'pv_estimate10': 0.0509, 'pv_estimate90': 0.2197}

### What version of Solcast Integration has the issue?

3.1.6

### What version of Home Assistant are you using?

Home Assistant 2023.8.2

### Logs debug/info/error output

```txt
This error originated from a custom integration.

Logger: custom_components.solcast_solar.solcastapi
Source: custom_components/solcast_solar/solcastapi.py:196
Integration: Solcast PV Forecast (documentation, issues)
First occurred: 07:53:16 (16 occurrences)
Last logged: 07:53:18

not using item {'period_end': '2023-08-25T03:00:00.0000000Z', 'pv_estimate': 0, 'pv_estimate10': 0, 'pv_estimate90': 0}
not using item {'period_end': '2023-08-25T03:30:00.0000000Z', 'pv_estimate': 0, 'pv_estimate10': 0, 'pv_estimate90': 0}
not using item {'period_end': '2023-08-25T05:00:00.0000000Z', 'pv_estimate': 0, 'pv_estimate10': 0, 'pv_estimate90': 0}
not using item {'period_end': '2023-08-25T05:30:00.0000000Z', 'pv_estimate': 0.0088, 'pv_estimate10': 0.0015, 'pv_estimate90': 0.0099}
not using item {'period_end': '2023-08-25T07:00:00.0000000Z', 'pv_estimate': 0.2092, 'pv_estimate10': 0.0509, 'pv_estimate90': 0.2197}
```

### Additional information

I dont think it effects the Solcast Data being produced but I thought I should let you Know

Regards
Dave Evans

**Comment by BettySwallocks** [comment]
Same issue, this is my top Sentry error logged - following.

**Comment by Ndrinta** [comment]
+1 here. Running 2023.8.2

Just noticed all those zeors! ~

I've got 2 API keys for Solcast and they both are reporting missing values.
I've tried reloading them a couple times without success... 
 
![image](https://github.com/oziee/ha-solcast-solar/assets/4704172/32d46b63-91d3-48bb-8f50-206dc1b0d192)

UPDATE: Whilst writing this comment I realized I had recently updated the Solcast integration to the latest version (v3.1.6) 
but not yet restarted HA; it's all good after the restart.
